--- generic_lmd_new_les_mpifort_64/WRFV2/configure.wrf	2020-03-27 12:52:35.000000000 +0100
+++ generic_lmd_new_les_mpifort_64_SAV/WRFV2/configure.wrf	2020-03-25 18:25:00.000000000 +0100
@@ -118,19 +118,19 @@
 OMP             =       # -openmp -fpp -auto
 SFC             =       ifort
 SCC             =       icc
-DM_FC           =       mpif90 -f90=$(SFC)
-DM_CC           =       mpicc -cc=$(SCC) -DMPI2_SUPPORT
+DM_FC           =       /usr/lib64/openmpi/1.6.5-ifort/bin//mpif90 -f90=$(SFC)
+DM_CC           =       /usr/lib64/openmpi/1.6.5-ifort/bin//mpicc -DMPI2_SUPPORT -cc=$(SCC) -DMPI2_SUPPORT
 FC              =        $(DM_FC)
 CC              =       $(DM_CC) -DFSEEKO64_OK 
 LD              =       $(FC)
-RWORDSIZE       =       $(NATIVE_RWORDSIZE)
-PROMOTION       =       -i4
-ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_FUNC
-CFLAGS_LOCAL    =       -w -O3 -ip
+RWORDSIZE       =       8 #$(NATIVE_RWORDSIZE)
+PROMOTION       =       -i4 -real-size 64
+ARCH_LOCAL      =       -DNEWPHYS -DNONSTANDARD_SYSTEM_FUNC
+CFLAGS_LOCAL    =       -w -O3 -mcmodel=large -shared-intel -ip
 LDFLAGS_LOCAL   =       -ip
 CPLUSPLUSLIB    =       
 ESMF_LDFLAG     =       $(CPLUSPLUSLIB)
-FCOPTIM         =       -O3
+FCOPTIM         =       -O3 -mcmodel=large -shared-intel
 FCREDUCEDOPT	=       $(FCOPTIM)
 FCNOOPT		=       -O0
 FCDEBUG         =       # -g $(FCNOOPT) -traceback
@@ -141,7 +141,7 @@
 FCBASEOPTS      =       -w -ftz -align all -fno-alias -fp-model precise $(FCDEBUG) $(FORMAT_FREE) $(BYTESWAPIO)
 MODULE_SRCH_FLAG =     
 TRADFLAG        =      -traditional
-CPP             =      /lib/cpp -C -P
+CPP             =      /lib/cpp  -P
 AR              =      ar
 ARFLAGS         =      ru
 M4              =      m4
@@ -183,6 +183,8 @@
                       -I$(WRF_SRC_ROOT_DIR)/share \
                       -I$(WRF_SRC_ROOT_DIR)/phys \
                       -I$(WRF_SRC_ROOT_DIR)/chem -I$(WRF_SRC_ROOT_DIR)/inc \
+                      -I$(WRF_SRC_ROOT_DIR)/../../../../LMDZ.COMMON/libo/CICLADifort/.config/inc \
+                      -I/opt/netcdf3/ifort/include \
                        \
                       
 REGISTRY        =    Registry
@@ -199,7 +201,10 @@
                      $(WRF_SRC_ROOT_DIR)/frame/pack_utils.o 
 
 LIB_EXTERNAL    = \
-                     $(WRF_SRC_ROOT_DIR)/external/io_netcdf/libwrfio_nf.a -L/opt/netcdf3/ifort/lib -lnetcdf    
+                     $(WRF_SRC_ROOT_DIR)/external/io_netcdf/libwrfio_nf.a \
+                     -L/opt/netcdf3/ifort/lib -lnetcdf -lnetcdff /opt/netcdf3/ifort/lib/libnetcdf.a \
+                     -L$(WRF_SRC_ROOT_DIR)/../../../../LMDZ.COMMON/libo/CICLADifort/.config/lib -llmdz \
+                     -lnetcdf  
 
 LIB             =    $(LIB_BUNDLED) $(LIB_EXTERNAL) $(LIB_LOCAL)
 LDFLAGS         =    $(OMP) $(FCFLAGS) $(LDFLAGS_LOCAL) 
